latexify/core.py

```diff
@@ -52,10 +52,10 @@ def visit_Module(self, node):
     return self.visit(node.body[0])
 
   def visit_FunctionDef(self, node):
-    name_str = r'\mathrm{' + str(node.name) + '}'
+    name_str = r'\operatorname{' + str(node.name) + '}'
```

Comment:
Revert this: operatorname is not supported by some printing systems.

Reply:
Done